projects/schematics/src/add-ssr/files/server.__typescriptExt__

```diff
@@ -1,20 +1,21 @@
-import 'zone.js/dist/zone-node';
+import { APP_BASE_HREF } from '@angular/common';
+import {
+  NgExpressEngineDecorator,
+  ngExpressEngine as engine,
+} from '@spartacus/setup/ssr';
 
-import { ngExpressEngine as engine } from '@nguniversal/express-engine';
-import { NgExpressEngineDecorator } from '@spartacus/setup/ssr';
-import * as express from 'express';
+import express from 'express';
 import { join } from 'path';
 
-import { AppServerModule } from './src/main.server';
-import { APP_BASE_HREF } from '@angular/common';
-import { existsSync } from 'fs';
+import { existsSync } from 'node:fs';
+import AppServerModule from './src/main.server';
 
 const ngExpressEngine = NgExpressEngineDecorator.get(engine);
 
 // The Express app is exported so that it can be used by serverless Functions.
 export function app() {
   const server = express();
-  const distFolder = join(process.cwd(), '<%= browserDistDirectory %>');
+  const distFolder = join(process.cwd(), 'dist/ng-17-test-ssr-module/browser');
```

Comment:
[MEDIUM]
Now it's hardcoded. It didn't work, when my app name was different than "ng-17-test-ssr-module". The error thrown while running `serve:ssr:<app-name>`:
```
Error: Failed to lookup view "index" in views directory "<base directory>/test-ng17-pr18228/dist/ng-17-test-ssr-module/browser"
```

Let's bring back the template variable. `<%  .... %>`